videoanalyser/app/src/main/java/com/example/videoanalyser/MainActivity.java

```diff
@@ -57,14 +58,16 @@ protected void onCreate(Bundle savedInstanceState) {
         filePath = findViewById(R.id.selectedFilePath);
         filePickerBtn = findViewById(R.id.fileSelectButton);
         analyzeBtn = findViewById(R.id.analyze);
+
         recognizer = TextRecognition.getClient();
+        mediaMetadataRetriever = new MediaMetadataRetriever();
 
         filePickerBtn.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
                 fileIntent = new Intent(Intent.ACTION_GET_CONTENT);
                 fileIntent.setType("*/*");
-                startActivityForResult(fileIntent, 10);
+                startActivityForResult(fileIntent, filePickerRequestCode);
```

Comment:
Can we name this FILE_PICKER_REQUEST_CODE to make it clear that it is a constant?

Reply:
(Styleguide reference: https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names )

Reply:
^ Assume that is the styleguide we want you to follow. And if we make comments inconsistent with that guide, feel free to push back :)